### PR TITLE
Updates ApostropheCMS information

### PR DESCRIPTION
**Pull request by abea**
## Summary

Proposed change:

This updates the description and quote for ApostropheCMS in the "Who's Using It?" section of the documentation site. Apostrophe is no longer housed within P'unk Avenue and has new URLs.

## Checklist

I've completed the checklist below to ensure I didn't forget anything. This makes reviewing this PR as easy as possible for the maintainers. And it gets this change released as soon as possible.

* [x] Proposed change helps towards [*purpose of this project*](https://github.com/mozilla/nunjucks/blob/master/CONTRIBUTING.md#purpose).
* [x] [*Documentation*](https://github.com/mozilla/nunjucks/tree/master/docs/) is added / updated to describe proposed change.
* [x] [*Tests*](https://github.com/mozilla/nunjucks/tree/master/tests) are added / updated to cover proposed change.
* [x] [*Changelog*](https://github.com/mozilla/nunjucks/blob/master/CHANGELOG.md) has an entry for proposed change (if user-facing fix or feature).

### CODE REVIEW

**Comment by abea** [comment]
The updated quote is from @boutell, who is now the CTO for Apostrophe.

**Comment by boutell** [comment]
Just confirming I agree with this request.

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "nunjucks",
   "description": "A powerful templating engine with inheritance, asynchronous control, and more (jinja2 inspired)",
-  "version": "3.2.3",
+  "version": "3.2.4",
```
Comment (ogonkov) [inline package.json:4]: You shouldn't change package version

**Review by ogonkov — CHANGES_REQUESTED** [review]

**Comment by ogonkov** [comment]
@abea  Changelog changes was OK )

**Review by ogonkov — APPROVED** [review]

**Comment by abea** [comment]
Hello. Is there any plan to merge this and release @ogonkov ?

**Comment by ogonkov** [comment]
I dont have permissions im afraid

**Comment by abea** [comment]
Closing due to inactivity. It is also outdated since ApostropheCMS is on version 3, which also uses Nunjucks. Tagging in @boutell in case the current team wants to resubmit with A3 information.

**Pull request closed without merging**